de-lib/src/main/java/org/iplantc/de/diskResource/client/ToolbarView.java

```diff
@@ -18,7 +18,6 @@
 import org.iplantc.de.diskResource.client.events.selection.RefreshFolderSelected.HasRefreshFolderSelectedHandlers;
 import org.iplantc.de.diskResource.client.events.selection.RenameDiskResourceSelected.HasRenameDiskResourceSelectedHandlers;
 import org.iplantc.de.diskResource.client.events.selection.RestoreDiskResourcesSelected.HasRestoreDiskResourceSelectedHandlers;
-import org.iplantc.de.diskResource.client.events.selection.SaveMetadataSelected.HasSaveMetadataSelectedEventHandlers;
```

Comment:
Gotcha. I took a quick peek at it and I think to do it without the EventBus would require some other refactors that would be outside the scope of this PR.  I think this is an appropriate solution.